ingestion-generated/setup.py

```diff
@@ -0,0 +1,39 @@
+#  Copyright 2021 Collate
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#  http://www.apache.org/licenses/LICENSE-2.0
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+import os
+
+from setuptools import setup, find_namespace_packages
+
+
+def get_long_description():
+    root = os.path.dirname(__file__)
+    with open(os.path.join(root, "README.md")) as f:
+        description = f.read()
+    return description
+
+
+setup(name="openmetadata-ingestion-generated",
+      version="0.1.0",
```

Comment:
can we keep the version the same as the ingestion package. This will cause confusion to associate with ingestion. It would be good to have same version for both package release.

Reply:
sure